interactive-calculator/src/containers/BugetSet/BudgetSet.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+export default ( { budget, split, setBudget, setSplit}) => {
+  return (
+    <div>
```

Comment:
I think if you check out the useContext hook you would get the same desired effect that you get by passing down these functions as props from the parent component. Its really cool!